test/wdio/scoped-basic/cmp.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { h, Fragment } from '@stencil/core';
```

Comment:
Non-blocking - `Fragment` isn't used and can be removed
```suggestion
import { h } from '@stencil/core';
```